ProcessMaker/Http/Controllers/Api/UserController.php

```diff
@@ -140,6 +140,78 @@ public function index(Request $request)
         return new ApiCollection($response);
     }
 
+    /**
+     * Display a listing of users and their task counts.
+     *
+     * @param \Illuminate\Http\Request $request
+     * @return \Illuminate\Http\Response
+     *
+     * @OA\Get(
+     *     path="/users_count_tasks",
+     *     summary="Returns all users and their total tasks",
+     *     operationId="getUsersTaskCount",
+     *     tags={"Users"},
+     *     @OA\Parameter(
+     *         name="filter",
+     *         in="query",
+     *         description="Filter results by string. Searches First Name, Last Name, Email, or Username.",
+     *         @OA\Schema(type="string"),
+     *     ),
+     *     @OA\Response(
+     *         response=200,
+     *         description="List of users with task counts",
+     *         @OA\JsonContent(
+     *             type="object",
+     *             @OA\Property(
+     *                 property="data",
+     *                 type="array",
+     *                 ref="#/components/schemas/users",
+     *             ),
+     *             @OA\Property(
+     *                 property="meta",
+     *                 type="object",
+     *                 ref="#/components/schemas/metadata",
+     *             ),
+     *         ),
+     *     ),
+     * )
+     */
+    public function getUsersTaskCount(Request $request)
+    {
+        if (!(Auth::user()->can('view-users') ||
```

Comment:
Please do this in the routes with `->middleware('can:view-users|create-processes|....`